gnd/src/main/java/com/google/android/gnd/ui/offlinebasemap/viewer/OfflineBaseMapViewerViewModel.java

```diff
@@ -42,46 +48,64 @@
 public class OfflineBaseMapViewerViewModel extends AbstractViewModel {
 
   @Hot(replays = true)
-  private final FlowableProcessor<OfflineBaseMapViewerFragmentArgs> argsProcessor =
-      BehaviorProcessor.create();
+  private final SingleSubject<OfflineBaseMapViewerFragmentArgs> fragmentArgs =
```

Comment:
Are we sure the view model won't be used more than once w/different args? If not it might be better to model this as an `Observable` (see previous comment).

Reply:
Switch to `PublishSubject`.